src/main/java/org/gridsuite/filter/server/FilterService.java

```diff
@@ -65,41 +65,17 @@ public class FilterService {
     private final FilterService self;
 
     public FilterService(final ScriptFilterRepository scriptFiltersRepository,
-                         final LineFilterRepository lineFilterRepository,
-                         final GeneratorFilterRepository generatorFilterRepository,
-                         final LoadFilterRepository loadFilterRepository,
-                         final ShuntCompensatorFilterRepository shuntCompensatorFilterRepository,
-                         final StaticVarCompensatorFilterRepository staticVarCompensatorFilterRepository,
-                         final BatteryFilterRepository batteryFilterRepository,
                          final BusBarSectionFilterRepository busBarSectionFilterRepository,
-                         final DanglingLineFilterRepository danglingLineFilterRepository,
                          final LccConverterStationFilterRepository lccConverterStationFilterRepository,
                          final VscConverterStationFilterRepository vscConverterStationFilterRepository,
```

Comment:
to be clean ?

Reply:
deleted. But we will delete the classes in another PR when we remove the criteria filter tables.